kie-based-services/src/main/java/io/elimu/serviceapi/config/ConfigAPIProcessVariableInitHelper.java

```diff
@@ -10,14 +11,18 @@
 
 public class ConfigAPIProcessVariableInitHelper extends ProcessVariableInitHelper {
 
-	public Map<String, Object> initVariables(ServiceRequest request, String client, Dependency dep, Properties config) {
+	public Map<String, Object> initVariables(ServiceRequest request, String client, Dependency dep, Properties config) throws TimeoutException{
 		Map<String, Object> serviceProperty = super.initVariables(dep);
 		String env = System.getProperty("configApiEnv");
 		String appName = config.getProperty("kie.project.appname");
 		if (appName == null || "".equals(appName.trim())) {
 			appName = dep.getArtifactId();
 		}
-		serviceProperty.putAll(new ConfigAPIUtil().getConfig(request, env, client, appName));
+		try {
+			serviceProperty.putAll(new ConfigAPIUtil().getConfig(request, env, client, appName));
+		} catch (TimeoutException e) {
```

Comment:
Suggestion, changing this catch block here to this:
`catch (TimeoutException e) { throw e; }`